### DISCUSSION TITLE
Does GPG key expose email address attached?

### DISCUSSION THREAD

**Discussion by Nicklaus-s** [community/community#53652] [Repositories] [answered]
### Select Topic Area

Question

### Body

Does publishing a signed commit via a GPG key expose the email address involved with the GPG key?
<img width="177" alt="image" src="https://user-images.githubusercontent.com/100449899/234177810-bc74db98-fc5b-4625-b31e-bf8c182c957c.png">

**Comment by airtower-luna** [comment, marked as answer]
Indirectly, yes. The commit itself contains exactly the same email addresses (author and committer) as it would without the signature. However the published public key(s) associated with your account contains a list (one or more) of names/addresses it is for.
